### DISCUSSION TITLE
问题反馈

### DISCUSSION THREAD

**Discussion by DanileLxp** [Tokinx/Wing#75] [讨论]
在前端笔记上传图片时，无法上传，并提示：undetined is not an object (evaluating'rv.headers.get')

**Comment by shmilyabyy** [comment]
同问

**Reply by Tokinx** [reply]
请检查上传目录权限是否有配置正确。
参考：https://www.danilelxp.com/note/5763

**Reply by DanileLxp** [reply]
20221214更新：权限能解决图片上传的问题，但奇怪的是中文名称也会出现这个提示且无法上传。

**Reply by Tokinx** [reply]
> 20221214更新：权限能解决图片上传的问题，但奇怪的是中文名称也会出现这个提示且无法上传。

一般都是数据库格式或者主机配置问题

**Reply by DanileLxp** [reply]
> > 20221214更新：权限能解决图片上传的问题，但奇怪的是中文名称也会出现这个提示且无法上传。
> 
> 一般都是数据库格式或者主机配置问题

暂时无解，影响不大，先放一边了。

**Reply by DanileLxp** [reply]
> > 20221214更新：权限能解决图片上传的问题，但奇怪的是中文名称也会出现这个提示且无法上传。
> 
> 一般都是数据库格式或者主机配置问题

20230111 新反馈：
- 安卓13，360浏览器下我的笔记全部消失，大佬你的最新四条笔记会消失；
- 目前文章置顶不生效，调用置顶函数依旧不生效，是被排除过滤了还是方法有问题？；
- 热力图的评论统计貌似只统计博主回复数；

**Comment by shmilyabyy** [comment]
我现在在手机上面显示是正常的时间。目前没有电脑。所以不能测试电脑是否正常。还有作者大大。我想知道笔记如何删除。

Jeff ***@***.***>于2022年12月3日 周六01:11写道：

**Reply by Tokinx** [reply]
<img width="750" alt="image" src="https://user-images.githubusercontent.com/4526339/205590428-0cf9b72b-cbba-4a19-9183-a5469fb6ed0c.png">

Archive可以将笔记归档（删除）

**Comment by shmilyabyy** [comment]
谢谢。我的问题都完美解决了。那个时间问题不知道怎么的就突然正常了。

Jeff ***@***.***>于2022年12月5日 周一16:33写道：
